### PR TITLE
Fixes clownnames.txt file into a A-Z 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changlings]
Makes the names that were added A->Z, like it was before, organizing!
[Hos is ling?]
It is how names are formatted in text files as far as I know, lets keep it nice and A->Z like!
Has no in game affects, helps my brain cope a lot, thats about it!

### CODE REVIEW

File: strings/names/clown.txt [hunk 1]
```diff
@@ -1,123 +1,122 @@
...
 Jingle
+Johnny Bigshoes
 Jo Jo Bobo Bo
+King of Klowns
 Ladybug Honks
+Laughing Man
+Loopy Lazarus
+Mime
+Misery Marvin
+Misery Marvin
```
Comment (tralezab) [inline strings/names/clown.txt]: is the double misery marvin on purpose?

File: strings/names/clown.txt [hunk 2]
```diff
@@ -1,123 +1,122 @@
...
+Fat Fingers
+Fishbreath
 Flop O'Honker
 Freckle
+Fretworks
+Galton
 Giggles
 Gigglesworth
 Goose McSunny
+GrabbyShoe Giving Gerald
```
Comment (SpaceManiac) [inline strings/names/clown.txt]: Should be "Grabby" and "Shoe Giving Gerald" separately

**Comment by DaxDupont** [comment]
Should've just done a sort instead of doing it manually Lmao

**Comment by ExcessiveUseOfCobblestone** [comment]
> old: 123 lines
> new: 122 lines

good luck lol

**Pull request merged**